include/tools.h

```diff
@@ -150,11 +150,11 @@ void sleep(unsigned int milliseconds);
  *
  * @param str The string to split.
  * @param delims A string of potential delimiters.
- * @param trimEmpty true if empty part must be trimmed from the result.
+ * @param dropEmpty true if empty part must be dropped from the result.
  * @param keepDelim true if delimiter must be included from the result.
  * @return a list of part (potentially containing delimiters)
  */
-std::vector<std::string> split(const std::string& str, const std::string& delims, bool trimEmpty=true, bool keepDelim = false);
+std::vector<std::string> split(const std::string& str, const std::string& delims, bool dropEmpty=true, bool keepDelim = false);
```

Comment:
The example suggests that each character of the string `delims` is a potential delimiter. It can be easily confused with the entire string being a single delimiter. I think we should mention this explicitly in the parameter docstring.